cts/cts-regression.in

```diff
@@ -188,10 +198,19 @@ while [ $# -gt 0 ] ; do
             valgrind="-v"
             shift
             ;;
-        scheduler|exec|pacemaker_remote|fencing|cli)
+        scheduler|exec|fencing|cli)
             add_test $1
             shift
             ;;
+        pacemaker_remote)
+            if [[ $REMOTE_ENABLED -eq 0 ]]; then
```

Comment:
Here, you're using double brackets but higher up you're using single brackets and at the beginning of the file you're using single brackets with -z.  I'd like to see one style of test if possible.

Reply:
Looking at the rest of the file, we're using double brackets just for bash-specific tests (regexes), so I'll switch to single here